graylog2-rest-client/src/main/java/org/graylog2/restclient/lib/ApiClientImpl.java

```diff
@@ -535,8 +547,11 @@ private void ensureAuthentication() {
                 try {
                     final AsyncHttpClient.BoundRequestBuilder requestBuilder = requestBuilderForUrl(url);
                     requestBuilder.addHeader(HttpHeaders.ACCEPT, mediaType.toString());
+                    // we need it for the APIException
+
+                    final Request request = requestBuilder.build();
                     if (LOG.isDebugEnabled()) {
-                        LOG.debug("API Request: {}", requestBuilder.build().toString());
+                        LOG.debug("API Request: {}", request.toString());
```

Comment:
Calling `toString()` is obsolete when using the parameterized logging syntax (and possibly counter-productive): http://slf4j.org/faq.html#string_contents